spec/factories.rb

```diff
@@ -257,6 +257,18 @@
     end
   end
 
+  factory :overdue_moderation_petition, :parent => :sponsored_petition do
+    moderation_threshold_reached_at (Site.moderation_overdue_in_days.days.ago - 5.minutes)
+  end
+
+  factory :nearly_overdue_moderation_petition, :parent => :sponsored_petition do
+    moderation_threshold_reached_at (Site.moderation_overdue_in_days.days.ago + 5.minutes)
+  end
+
+  factory :recently_in_moderation_petition, :parent => :sponsored_petition do
+    moderation_threshold_reached_at (Time.current)
+  end
+
```

Comment:
These names are a little unwieldy - maybe better as traits on the `:sponsored_petition` factory